opencti-platform/opencti-graphql/src/database/file-storage-helper.ts

```diff
@@ -114,3 +135,42 @@ export const deleteAllBucketContent = async (context: AuthContext, user: AuthUse
   // Once all files are deleted, then bucket can be removed.
   // await deleteBucket();
 };
+
+/**
+ * Move all file from source entity to target entity and then delete cleanup directories on S3.
+ * @param context
+ * @param user
+ * @param sourceEntity
+ * @param targetEntity
+ */
+export const moveAllFilesFromEntityToAnother = async (context: AuthContext, user: AuthUser, sourceEntity: BasicStoreBase, targetEntity: BasicStoreBase) => {
+  const updatedXOpenctiFiles: Partial<S3File> [] = [];
+
+  for (let folderI = 0; folderI < ALL_MERGEABLE_FOLDERS.length; folderI += 1) {
+    try {
+      const sourcePath = `${ALL_MERGEABLE_FOLDERS[folderI]}/${sourceEntity.entity_type}/${sourceEntity.internal_id}`;
+      const targetPath = `${ALL_MERGEABLE_FOLDERS[folderI]}/${targetEntity.entity_type}/${targetEntity.internal_id}`;
+      const importFilesToMove = await allFilesForPaths(context, user, [sourcePath]);
+
+      for (let fileI = 0; fileI < importFilesToMove.length; fileI += 1) {
+        const sourceFileDocument = importFilesToMove[fileI];
+        const sourceFileS3Id = `${sourcePath}/${sourceFileDocument.name}`;
+        const targetFileS3Id = `${targetPath}/${sourceFileDocument.name}`;
+        logApp.info(`[FILE STORAGE] Moving from ${sourceFileS3Id} to :${targetFileS3Id}`);
+        const newFile = await copyFile(sourceFileS3Id, targetFileS3Id, sourceFileDocument, targetEntity.internal_id);
+        if (newFile) {
+          await deleteFile(context, user, sourceFileS3Id); // TODO to be removed ? This will be done by merge delete no ?
+
+          const newFileForEntity = storeFileConverter(user, newFile);
+          logApp.info(' [FILE STORAGE] Adding x_opencti_file', { newFileForEntity });
+          updatedXOpenctiFiles.push(newFileForEntity);
+        }
+      }
+    } catch (err) {
+      logApp.error(`[FILE STORAGE] Merge of files failed in ${ALL_MERGEABLE_FOLDERS[folderI]}`, { cause: err, user_id: user.id, sourceEntity, targetEntity });
```

Comment:
```suggestion
      logApp.error(`[FILE STORAGE] Merge of files failed`, { cause: err, user_id: user.id, sourceEntity, targetEntity, folder: ${ALL_MERGEABLE_FOLDERS[folderI]} });
```
to avoid generated content in error message ; this ease automated log parsing